eanalyze_staging.sql

```diff
@@ -0,0 +1,1483 @@
+  KEY `shipment_map_id` (`shipment_map_id`) /*!80000 INVISIBLE */,
+  KEY `sample_id` (`sample_id`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Table structure for table `response_result_vl`
+--
+
+DROP TABLE IF EXISTS `response_result_vl`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `response_result_vl` (
+  `shipment_map_id` int NOT NULL,
+  `sample_id` varchar(45) NOT NULL,
+  `reported_viral_load` varchar(255) DEFAULT NULL,
+  `calculated_score` varchar(45) DEFAULT NULL,
+  `is_tnd` varchar(45) DEFAULT NULL,
+  `created_by` varchar(45) DEFAULT NULL,
+  `created_on` datetime DEFAULT NULL,
+  `updated_by` varchar(45) DEFAULT NULL,
+  `updated_on` datetime DEFAULT NULL,
+  PRIMARY KEY (`shipment_map_id`,`sample_id`),
+  CONSTRAINT `response_result_vl_ibfk_1` FOREIGN KEY (`shipment_map_id`) REFERENCES `shipment_participant_map` (`map_id`) ON DELETE CASCADE
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Table structure for table `scheme_list`
+--
+
+DROP TABLE IF EXISTS `scheme_list`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `scheme_list` (
+  `scheme_id` varchar(10) NOT NULL,
+  `scheme_name` varchar(255) NOT NULL,
+  `response_table` varchar(45) DEFAULT NULL,
+  `reference_result_table` varchar(45) DEFAULT NULL,
+  `attribute_list` varchar(255) DEFAULT NULL,
+  `status` varchar(255) DEFAULT NULL,
+  PRIMARY KEY (`scheme_id`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Table structure for table `shipment`
+--
+
+DROP TABLE IF EXISTS `shipment`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `shipment` (
+  `shipment_id` int NOT NULL AUTO_INCREMENT,
+  `shipment_code` varchar(255) NOT NULL,
+  `scheme_type` varchar(10) DEFAULT NULL,
+  `shipment_date` date DEFAULT NULL,
+  `lastdate_response` date DEFAULT NULL,
+  `distribution_id` int NOT NULL,
+  `number_of_samples` int DEFAULT NULL,
+  `number_of_controls` int NOT NULL,
+  `response_switch` varchar(255) NOT NULL DEFAULT 'off',
+  `max_score` int DEFAULT NULL,
+  `average_score` varchar(255) DEFAULT '0',
+  `shipment_comment` text,
+  `created_by_admin` varchar(255) DEFAULT NULL,
+  `created_on_admin` datetime DEFAULT NULL,
+  `updated_by_admin` varchar(255) DEFAULT NULL,
+  `updated_on_admin` datetime DEFAULT NULL,
+  `status` varchar(255) NOT NULL DEFAULT 'pending',
+  `follows_up_from` int DEFAULT NULL,
+  `is_official` int DEFAULT '1',
+  PRIMARY KEY (`shipment_id`),
+  KEY `scheme_type` (`scheme_type`),
+  KEY `distribution_id` (`distribution_id`),
+  KEY `shipment_ibfk_3_idx` (`follows_up_from`),
+  KEY `shipment_id` (`shipment_id`),

```

Comment:
This key is unnecessary as shipment_id is already the primary key for this table. Please remove this?

Reply:
okay